CMakeLists.txt

```diff
@@ -536,6 +539,15 @@ if(UA_ENABLE_TPM2_SECURITY)
     list(APPEND open62541_LIBRARIES ${TPM2_LIB})
 endif()
 
+if(UA_ENABLE_LWS)
+    find_package(libwebsockets REQUIRED)
+#    message(LWS Version: ${LIBWEBSOCKETS_VERSION})
```

Comment:
If you need LWS >= 4.3, then enable the fatal error.
Otherwise remove the comment.